llvm/lib/Target/AIE/AIEInterBlockScheduling.cpp

```diff
@@ -233,6 +235,7 @@ namespace {
 /// into the appropriate blockstate region.
 /// TimedRegion is built one bundle at the time
 class PipelineExtractor : public PipelineScheduleVisitor {
+  InterBlockScheduling &InterBlock;
```

Comment:
We seem to only use `InterBlock.getSelectedAltDescs()`. Maybe just have a `AlternateDescriptors` member?